fvm/systemcontracts/system_contracts.go

```diff
@@ -153,6 +159,9 @@ var (
 	stakingContractAddressMainnet = flow.HexToAddress("8624b52f9ddcd04a")
 	// stakingContractAddressTestnet is the address of the FlowIDTableStaking contract on Testnet and Canary
 	stakingContractAddressTestnet = flow.HexToAddress("9eca2b38b18b5dfe")
+
+	flowFeesContractAddressMainnet = flow.HexToAddress("f919ee77447b7497")
+	flowFeesContractAddresTestnet  = flow.HexToAddress("912d5440f7e3769e")
 )
```

Comment:
Why do we need to include these fee contracts into the system_contracts?

Reply:
In the [core contract changes](https://github.com/onflow/flow-core-contracts/pull/279) a new fee deduction function was added to the `FlowFees` contract (the old function was kept in place so that we can switch to the new calculation without braking the old one once the contract is updated). 

Since the fee deduction is now in FlowFees, we need the address of FlowFees to be known.

Reply:
yeah but system contract is mostly used for epoch stuff, so they had no choice to hard code them here, I wonder if it should be computed on the fly or at least be in a different place

Reply:
I think it will always be the 4th address (i would have to verify). Would it be better to use that?